pages/events/codingclub.md

```diff
@@ -0,0 +1,16 @@
+---
+layout : base-layout.ejs
+title : CodingClub
+tags: ["event"]
+permalink : /events/Coding Club/

```

Comment:
it should be sluggish cased (all lowercase, words separated by a hyphen), `/events/coding-club`